### DISCUSSION TITLE
Playlist View - follow up

### DISCUSSION THREAD

**Discussion by treboa** [TT-ReBORN/Georgia-ReBORN#156] [Q&A] [answered]
Sorry @TT-ReBORN but I’m still having trouble with the adjustment I want to make in the Playlist view track row (with header showing, default style). What I want to show is, for example:
        01 - Pharaoh's Dance
Not: 01. Pharaoh's Dance
All my albums are tagged this way. I don’t want to alter anything else in the track row and I did this easily in the Library, tree view, by right click > configure and just changed it there but I’m going round in circles attempting the same result in Playlist view. Help!

**Comment by TT-ReBORN** [comment, marked as answer]
Open `foobar2000\profile\georgia-reborn\scripts\Base\gr-settings.js` and go to line ~1886:

Replace:
tf.vinyl_track = `$if2(${tf.vinyl_side}[${tf.vinyl_tracknum}]. ,[%tracknumber%. ])`;
to:
tf.vinyl_track = `$if2(${tf.vinyl_side}[${tf.vinyl_tracknum}] - ,[%tracknumber% - ])`;

Now the title formatting should be displayed as `01 - Pharaoh's Dance` instead of `01. Pharaoh's Dance`.

-TT

**Reply by treboa** [reply]
Thanks so much for that, I looked everywhere! Really enjoying your theme and foobar in general - playback and streaming so much better than MM which I’ve now ditched altogether. Thanks once again for your help.

treboa
